Update docs for EMQX open source limitations (#2574)
emqtt/emq-docs-cn: en_US/dashboard/rules.md

To create and manage the rules, you can access the **Rules** page by clicking **Integration** -> **Rules** from the left menu.

## Create Rules

To create a Rule, click **Create** on the upper right corner of the Rules page. You can also create a Rule quickly by clicking **Create Rule** from the **Action** column of an existing connector on the Connector page. 

### SQL Editor

The Create Rule page features an SQL editor where you can define rule logic using SQL statements. These statements enable real-time data manipulation, including querying, filtering, transforming, and enriching data exchanged between clients or systems.

To identify and organize rules, you'll provide a Rule ID and optional Note. A unique Rule ID is automatically generated, but you can also specify your own. The Note field allows for brief descriptions to clarify the rule's purpose.

> By clicking the **SQL Examples** below the SQL Editor, you can see some common SQL examples on the right side. You can quickly create rule by refercing to one of the following SQL examples:

Reviewer: ```suggestion
By clicking the **SQL Examples** below the SQL Editor, you can see some common SQL examples on the right side. You can quickly create rule by referring to one of the following SQL examples:
```

<img src="./assets/rule-sql.png" alt="image" style="zoom:67%;" />

> The default SQL statement is `SELECT * FROM "t/#"`, which means that when the client publish a message to the topic `t/#`, the Rule Engine will take out all the data under the current event. 

Reviewer: ```suggestion
The default SQL statement is `SELECT * FROM "t/#"`, which means that when the client publishes a message to the topic `t/#`, the Rule Engine will take out all the data under the current event. 
```

The `SELECT` keyword can get all the fields in the message, for example, if you want to get the current message's `Payload`, it can be modified to `SELECT payload from "t/#"`. The data can be processed and transformed using [built-in functions](../data-integration/rule-sql-builtin-functions.md). 

> The `FROM` keyword followed by data source. That is, the event topic, the **Try It Out** area below the SQL Editor can view all the optional data source events, using the `WHERE` keyword, you can add conditional judgment, for more detailed SQL syntax usage, see [SQL syntax and examples](./../data-integration/rule-sql-syntax.md).

Reviewer: ```suggestion
The `FROM` keyword is followed by one or more data sources. To see possible event topics, the **Try It Out** area below the SQL Editor can view all the optional data source events. Using the `WHERE` keyword, you can add conditional filtering. For more detailed SQL syntax usage, see [SQL syntax and examples](./../data-integration/rule-sql-syntax.md).
```

### Try It Out

> When the SQL statement is completed, you can click the **Try It Out** toggle switch at the bottom of the page to run the SQL test or Rule test in the this area.

Reviewer: ```suggestion
When the SQL statement is completed, you can click the **Try It Out** toggle switch at the bottom of the page to run the Rule test in this area.
```

- Subscribed ($events/session_subscribed)
- Unsubscribed ($events/session_unsubscribed)

The corresponding data source needs to be consistent with the SQL statement in the editor. When you need to use some message events to get data, you need to fill in the corresponding event topic (the content in the brackets) into the SQL statement after the `FROM` keyword, and rules support the use of multiple events. For more details about data source and event explanations, see [SQL Data Source and Fields](../data-integration/rule-sql-events-and-fields.md).

> You can also test the rule in the testing area. For detailed testing steps, you can see [Test Rule](../data-integration/rule-get-started.md#test-rule) for details.

Reviewer: Is this duplicated?  The current section seems to be about rule testing already.

Author: There are two functions with two separate tabs under the "Try It Out" in Dashboard now: SQL testing and Rule testing.

<img src="./assets/rule-test.png" alt="image" style="zoom:67%;" />

### Action Outputs

> After completing the editing of the SQL statement and the rule debugging, you can get output data that meets our requirements. At this time, you can select the action that needs to be executed after the rule is triggered in the **Action Outputs** tab on the right side of the page. Click the **Add Action** button, and you can select the type of the actions, including 2 built-in actions and various Sink/Source to continue to process the output data of the rule. For more detailed information about creating actions, see [Add Action](../data-integration/rule-get-started.md#add-action).

Reviewer: ```suggestion
After completing the editing of the SQL statement and the rule debugging, you can get output data that meets your requirements. At this time, you can select the action that needs to be executed after the rule is triggered in the **Action Outputs** tab on the right side of the page. Click the **Add Action** button, and you can select the type of the actions, including 2 built-in actions and various Sink/Source to continue to process the output data of the rule. For more detailed information about creating actions, see [Add Action](../data-integration/rule-get-started.md#add-action).
```
